schemas/littlehorse/service.proto

```diff
@@ -355,6 +373,35 @@ message PutTaskDefRequest {
   ReturnType return_type = 3;
 }
 
+// Creates a new StructDef.
+message PutStructDefRequest {
+  enum AllowedStructDefUpdateType {
+    // No updates are allowed.
+    NO_SCHEMA_UPDATES = 0;
+
+    // Allowed to make fully compatible (both backward-and-forward compatible)
+    // changes to the `struct_def` in this request.
+    FULLY_COMPATIBLE_SCHEMA_UPDATES = 1;
+  }
+
+  // The name of the StructDef.
+  string name = 1;
+
+  // The descripton of the StructDef.
+  optional string description = 2;
```

Comment:
This makes me happy. We should update all of the other metadata rpc's and `Getable`s as well.